detectron2/modeling/proposal_generator/proposal_utils.py

```diff
@@ -129,36 +129,35 @@ def find_top_rpn_proposals(
         results.append(res)
     return results
 
-
-def add_ground_truth_to_proposals(gt_boxes, proposals):
+def add_ground_truth_to_proposals(gt: List[Instances], proposals: List[Instances]):
```

Comment:
To maintain backward compatibility, we should allow the function to take `Union[List[Instances], List[Boxes]]`